pkgs/applications/window-managers/weston/default.nix

```diff
@@ -1,21 +1,25 @@
-{ lib, stdenv, fetchurl
+{ lib, stdenv, fetchFromGitLab
 , meson, ninja, pkg-config, python3, wayland-scanner
 , cairo, colord, dbus, lcms2, libGL, libXcursor, libdrm, libevdev, libinput
 , libjpeg, seatd, libxcb, libxkbcommon, mesa, mtdev, pam, udev, wayland
 , wayland-protocols
 , pipewire ? null, pango ? null, libunwind ? null, freerdp ? null, vaapi ? null
 , libva ? null, libwebp ? null, xwayland ? null
 # beware of null defaults, as the parameters *are* supplied by callPackage by default
-, buildDemo ? false
+, buildDemo ? true
+, buildRemoting ? true, gst_all_1
 }:
 
 stdenv.mkDerivation rec {
   pname = "weston";
-  version = "11.0.0";
+  version = "11.0.1";
 
-  src = fetchurl {
-    url = "https://gitlab.freedesktop.org/wayland/weston/-/releases/${version}/downloads/weston-${version}.tar.xz";
-    sha256 = "078y14ff9wmmbzq314f7bq1bxx0rc12xy4j362n60iamr56qs4x6";
+  src = fetchFromGitLab {
```

Comment:
Let's keep using the official tarballs unless there's a compelling reason to change.

Reply:
Yep, that looks right — it's the URL in the release announcement.